src/modules/sensors/sensor_params.c

```diff
@@ -213,6 +213,36 @@ PARAM_DEFINE_FLOAT(SENS_BOARD_Z_OFF, 0.0f);
  */
 PARAM_DEFINE_INT32(SENS_EN_THERMAL, -1);
 
+/**
+* Driver level notch frequency for gyro
+*
+* The center frequency for the 2nd order notch filter on the gyro driver.
+* This filter can be enabled to avoid feedback amplification of structural resonances at a specific frequency.
+* This only affects the signal sent to the controllers, not the estimators. 0 disables the filter.
+* See "IMU_GYRO_NOTCH_W" to set the bandwidth of the filter.
+*
+* @min 0
+* @max 1000
+* @unit Hz
+* @reboot_required true
+* @group Sensors
+*/
+PARAM_DEFINE_FLOAT(IMU_GYRO_NOTCH_F, 0.0f);
+
+/**
+* Driver level notch bandwidth for gyro
+*
+* The frequency width of the stop band for the 2nd order notch filter on the gyro driver.
+* See "IMU_GYRO_NOTCH_F" to activate the filter and to set the notch frequency.
+*
+* @min 0
+* @max 100
+* @unit Hz
+* @reboot_required true
+* @group Sensors
+*/
+PARAM_DEFINE_FLOAT(IMU_GYRO_NOTCH_W, 20.0f);
```

Comment:
Probably overthinking this, but another option could be to use NF for "notch filter" and then have room to be a bit more descriptive with frequency and bandwidth.

`IMU_GYRO_NOTCH_F -> IMU_GYRO_NF_FREQ`
`IMU_GYRO_NOTCH_W -> IMU_GYRO_NF_BW`

Reply:
Good idea!